modules/serverless-knative-kafka-trigger-metrics.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/admin-metrics/serverless-admin-metrics.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="serverless-knative-kafka-trigger-metrics_{context}"]
+= Knative Kafka trigger metrics
+
+You can use the following metrics to debug and visualize the performance of Kafka triggers.
+
+[cols=5*,options="header"]
+|===
+|Metric name
+|Description
+|Type
+|Tags
+|Unit
+
+|`event_count_1_total{job="kafka-broker-dispatcher-sm-service", namespace="knative-eventing"}`
+|Number of events dispatched by a trigger to a subscriber
+|Counter
+a|
+* `consumer_name` (Trigger name)
+* `namespace_name` (trigger namespace)
+* `name` (Broker name)
+* `event_type` (event type)
+* `response_code` (HTTP response code returned by the Trigger subscriber service)
+* `response_code_class` (HTTP response code class returned by the Trigger subscriber service: 2xx, 3xx, 4xx, 5xx)
```

Comment:
Not sure about what is correct or not here, but consider the previous note about API objects per [repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting). E.g., backticks if init-capped like `Trigger`, lowercase if a general reference to a "trigger". Same for "Broker" in here.

Same comments for the remainder of this table.

Reply:
The correct word should be the `trigger`.